lib/appium.js

```diff
@@ -177,6 +177,11 @@ class AppiumDriver extends BaseDriver {
     // It is not recommended to access this property directly from the outside
     this.pendingDrivers = {};
 
+    // initialize security modes
```

Comment:
these are not used

Reply:
Yeah, I need to put them into BaseDriver instead, oops.